src/main/java/meteordevelopment/meteorclient/systems/modules/movement/SafeWalk.java

```diff
@@ -78,7 +78,6 @@ private boolean blockFilter(Block block) {
         if (block instanceof AbstractRailBlock) return true;
         if (block instanceof TrapdoorBlock) return true;
         if (block instanceof PowderSnowBlock) return true;
-        if (block instanceof AbstractCauldronBlock) return true;
-        return false;
+        return block instanceof AbstractCauldronBlock;
```

Comment:
Personally, I think that the way it was before should be preferred because of easy adaptability